src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/InnvilgeVedtak/InntektsperiodeValg.tsx

```diff
@@ -98,20 +84,20 @@ const InntektsperiodeValg: React.FC<Props> = ({
     };
 
     return (
-        <>
-            <TittelContainer lesevisning={!behandlingErRedigerbar}>
+        <div className={className}>
```

Comment:
Hva er grunnen til å bruke flere Grids med faste størrelser for hver rad vs. å ha hele tingen i en grid med flere rader? I siste tilfelle kan man ha mer dynamiske størrelser fordi om en kolonne utvides vil det utvides for alle periodene og overskriftene også